test/binary/readelf/empty-archive.a

```diff
@@ -0,0 +1 @@
+!<arch>
```

Comment:
Could you please put it in test/readelf/ the binary folder name is for binary-rpms, it is bit misledading I suppose.

Reply:
Yep.